standard/lexical-structure.md

```diff
@@ -65,16 +76,17 @@ The productions for *simple_name* ([§12.8.4](expressions.md#1284-simple-names))
 >
 > *end example*
 
-If a sequence of tokens can be parsed (in context) as a *simple_name* ([§12.8.4](expressions.md#1284-simple-names)), *member_access* ([§12.8.7](expressions.md#1287-member-access)), or *pointer_member_access* ([§23.6.3](unsafe-code.md#2363-pointer-member-access)) ending with a *type_argument_list* ([§8.4.2](types.md#842-type-arguments)), the token immediately following the closing `>` token is examined, to see if it is
+If a sequence of tokens can be parsed, in context, as one of the disambiguated productions
+including an optional *type_argument_list* ([§8.4.2](types.md#842-type-arguments)), then
+the token immediately following the closing `>` token shall be examined and if it is:
```

Comment:
I think our tool will end up handling this better if we use soft breaks (no semantic change here):

```suggestion
If a sequence of tokens can be parsed, in context, as one of the disambiguated productions including an optional *type_argument_list* ([§8.4.2](types.md#842-type-arguments)), then the token immediately following the closing >` token shall be examined and if it is:
```

Reply:
I’ll join these in the next commit clobbering the stray §.